js/view.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+function hideElement(elementId){
+    var element = document.getElementById(elementId);
+    if(element){
+        element.setAttribute("hidden", '');
```

Comment:
Too many new attributes. Better use attribute 'data-', which is used to store additional information about the item.